### PR TITLE
Some final minor fixes to the course.

### PR DESCRIPTION
**Pull request by lbdreyer**
A few more fixes I found when running through the course from start to finish...

### CODE REVIEW

File: course_content/iris_course/2.Loading_and_Saving.ipynb [hunk 1]
```diff
@@ -212,27 +212,21 @@
     "# space for user code ..."
    ]
   },
-  {
-   "cell_type": "markdown",
-   "metadata": {},
-   "source": [
-    "2\\. Check whether this cube has lazy data? Print the minimum, maximum, mean and standard deviation of the cube's data. Does the cube still have lazy data?"
```
Comment (lbdreyer) [inline course_content/iris_course/2.Loading_and_Saving.ipynb]: The section on lazy data has been moved to the advanced section

File: course_content/iris_course/3.Subcube_Extraction.ipynb [hunk 2]
```diff
@@ -534,7 +534,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "1\\. Load the `sea_water_potential_temperature` cube from the file in ``iris.sample_data_path('atlantic_profiles.nc')`` and print the cube list. Store these cubes in a variable called `cube`."
```
Comment (lbdreyer) [inline course_content/iris_course/3.Subcube_Extraction.ipynb]: this must have just been a copy and past error as we are loading in a cube not a cubelist.

File: course_content/iris_course/5.Cube_Plotting.ipynb [hunk 3]
```diff
@@ -223,7 +223,6 @@
     "    <b><font color=\"brown\">Exercise: </font></b>\n",
     "    <p>Plot the Iris equivalent of the colour blockplot method \n",
     "      <a href=\"https://matplotlib.org/api/_as_gen/matplotlib.pyplot.pcolormesh.html\">matplotlib.pyplot.pcolormesh</a>    for the first timestep of the 'airtemps' data, i.e. <b><font face=\"courier\" color=\"black\">airtemps[0]</font></b>.</p> \n",
-    "    <p>Zoom in on a small region so you can see the individual data points.</p>\n",
```
Comment (lbdreyer) [inline course_content/iris_course/5.Cube_Plotting.ipynb]: I've removed this bit as we haven't introduced the fact that iris create cartopy geoaxes yet. That is done in the section after this exercise.

File: course_content/iris_course/3.Subcube_Extraction.ipynb [hunk 4]
```diff
@@ -494,9 +494,9 @@
    "metadata": {},
    "source": [
     "<div class=\"alert alert-block alert-warning\">\n",
-    "    <b><font color='brown'>Exercise: </font></b>Load in the `air_potential_temperature` cube from the file in <pre>iris.sample_data_path('uk_hires.pp')</pre> Iterate over this cube such that you print out subcubes of the following dimensionality:\n",
-    "    <pre>air_potential_temperature / (K)     (time: 3; grid_latitude: 204; grid_longitude: 187)</pre>\n",
-    "    Do this first using `slices` then again using `slices_over`\n",
+    "    <b><font color='brown'>Exercise: </font></b>Load in the `air_potential_temperature` cube from the file in <b><font face=\"courier\" color=\"black\">iris.sample_data_path('uk_hires.pp')</font></b> Iterate over this cube such that you print out subcubes of the following dimensionality:\n",
+    "    <p><b><font face=\"courier\" color=\"black\">air_potential_temperature / (K)     (time: 3; grid_latitude: 204; grid_longitude: 187)</font></b></p>\n",
```
Comment (pp-mo) [inline course_content/iris_course/3.Subcube_Extraction.ipynb]: This has also lost a linebreak, or at least a full stop, between the highlighted code + "Iterate over..."
( But the "Do this first..." bit still works as it did )

File: course_content/iris_course/5.Cube_Plotting.ipynb (hunk 3 above)
Comment (pp-mo) [reply course_content/iris_course/5.Cube_Plotting.ipynb]: I'm a bit conflicted about that :  I wanted this to demonstrate what we just said about how a blockplot is different from contouring and why you would use it.  And for that, you really need to see the individual cells ...

Could we "zoom in" by plotting only a small section of the data, instead ??

File: course_content/iris_course/5.Cube_Plotting.ipynb [hunk 5]
```diff
@@ -384,10 +383,17 @@
...
     "\n",
     "In this section we learnt:\n",
-    "* `iris.plot` and `iris.qplt` are used to create plots of the data stored in iris Cubes.\n",
-    "* `iris.qplt` automatically adds a plot title, axis titles, and a colour bar when appropriate\n",
+    "* `iris.plot` and `iris.quickplot` are used to create plots of the data stored in iris Cubes.\n",
+    "* `iris.quickplot` automatically adds a plot title, axis titles, and a colour bar when appropriate\n",
     "* Cartopy is used to project the Cube's data on to different map projections."
    ]
+  },
+  {
```
Comment (pp-mo) [inline course_content/iris_course/5.Cube_Plotting.ipynb]: An extra blank cell has crept back in.
This keeps happening, perhaps we should just leave them ?

File: course_content/iris_course/5.Cube_Plotting.ipynb [hunk 6]
```diff
@@ -384,10 +383,17 @@
     "## 5.4 Section Summary: Cube Plotting<a id='summary'></a>\n",
     "\n",
     "In this section we learnt:\n",
-    "* `iris.plot` and `iris.qplt` are used to create plots of the data stored in iris Cubes.\n",
-    "* `iris.qplt` automatically adds a plot title, axis titles, and a colour bar when appropriate\n",
+    "* `iris.plot` and `iris.quickplot` are used to create plots of the data stored in iris Cubes.\n",
```
Comment (pp-mo) [inline course_content/iris_course/5.Cube_Plotting.ipynb]: :+1: good spot !!

File: course_content/iris_course/6.Data_Processing.ipynb [hunk 7]
```diff
@@ -16,7 +16,7 @@
     "## Iris introduction course\n",
     "# 6. Data Processing\n",
     "\n",
-    "**Learning Outcome**: by the end of this section, you will be able to apply arithmetic and statistical operations on cube data.\n",
+    "**Learning Outcome**: by the end of this section, you will be able to apply arithmetic and statistical operations on the data of a Cube.\n",
```
Comment (pp-mo) [inline course_content/iris_course/6.Data_Processing.ipynb]: I think "the data of a Cube" is still not quite right.
( after all, we definitely *don't* mean `cube.data` )
Maybe just "perform arithmetic and statistical operations on a Cube" ?

File: course_content/iris_course/6.Data_Processing.ipynb [hunk 8]
```diff
@@ -236,7 +236,7 @@
     "        <b><font face=courier color=\"black\">product = e1 * e1.coord('latitude')</font></b>, \n",
     "        the data array content comes from multiplying <font face=courier color=\"black\">e1<b>.data</b></font> \n",
     "        times <font face=courier color=\"black\">e1.coord('latitude')<b>.points</b></font>.</p>\n",
-    "    <p>What happens if you simply multiply those two arrays, and are the values the same ?</p>\n",
+    "    <p>What happens if you simply multiply those two arrays ? Are the values the same ?</p>\n",
```
Comment (pp-mo) [inline course_content/iris_course/6.Data_Processing.ipynb]: :+1: !

File: course_content/iris_course/8.Final_Exercise.ipynb [hunk 9]
```diff
@@ -145,7 +145,9 @@
    "source": [
     "#### Part 4\n",
     "\n",
-    "Add a \"decade\" coordinate to the loaded cube using your function and the coord categorisation module."
+    "Add a \"decade\" coordinate to the loaded cube using your function and the coord categorisation module.\n",
+    "\n",
+    "(Hint: Look at the documentation for <a href='https://scitools.org.uk/iris/docs/v1.9.0/html/iris/iris/coord_categorisation.html#iris.coord_categorisation.add_categorised_coord'>iris.coord_categorisation.add_categorised_coord</a> to see how a function can be used to create a new coordinate.)"
```
Comment (pp-mo) [inline course_content/iris_course/8.Final_Exercise.ipynb]: I don't think "to see how a function can be used to create a new coordinate" reads very easily.
Suggest maybe : "<link>, which can be used to create a new categorised coordinate from such a function."

File: course_content/iris_course/5.Cube_Plotting.ipynb (hunk 3 above)
Comment (lbdreyer) [reply course_content/iris_course/5.Cube_Plotting.ipynb]: Addressed in #168

File: course_content/iris_course/1.The_Iris_Cube.ipynb [hunk 10]
```diff
@@ -431,7 +431,7 @@
    "metadata": {},
    "source": [
     "<div class=\"alert alert-block alert-warning\">\n",
-    "<b><font color=\"brown\">Exercise: </font></b>Take a look at the <a href=http://cfconventions.org/standard-names.html> CF standard name table</a>) and try renaming the cube to the an accepted name.\n",
+    "<b><font color=\"brown\">Exercise: </font></b>Take a look at the (<a href=http://cfconventions.org/standard-names.html> CF standard name table</a>) and try renaming the cube to an accepted name.\n",
     "</div>"
```
[the remark this answers is not in the document]
Comment (lbdreyer) [reply course_content/iris_course/1.The_Iris_Cube.ipynb]: Oh that does make more sense!

File: course_content/iris_course/2.Loading_and_Saving.ipynb [hunk 11]
```diff
@@ -200,7 +200,7 @@
     "\n",
     "## 2.3 Section Review Exercise<a id='exercise_2'></a>\n",
     "\n",
-    "1\\. Load the file in `iris.sample_data_path('atlantic_profiles.nc')`, as in Exercise 1, but this time use `iris.load_cube` to load in the `sea_water_potential_temperature` cube only."
+    "1\\. Load the file in `iris.sample_data_path('atlantic_profiles.nc')` using `iris.load_cube` to load in the `sea_water_potential_temperature` cube only."
    ]
```
[the remark this answers is not in the document]
Comment (lbdreyer) [reply course_content/iris_course/2.Loading_and_Saving.ipynb]: I am a big fan of commas so I will never no to more commas!

File: course_content/iris_course/2.Loading_and_Saving.ipynb [hunk 12]
```diff
@@ -212,27 +212,21 @@
...
-  },
   {
    "cell_type": "code",
    "execution_count": null,
    "metadata": {},
    "outputs": [],
    "source": [
-    "# space for user code ..."
+    "# SAMPLE SOLUTION\n",
+    "%load solutions/iris_exercise_2.3a"
```
[the remark this answers is not in the document]
Comment (lbdreyer) [reply course_content/iris_course/2.Loading_and_Saving.ipynb]: Accidental, sorry! Will fix.

File: course_content/iris_course/5.Cube_Plotting.ipynb (hunk 5 above)
Comment (lbdreyer) [reply course_content/iris_course/5.Cube_Plotting.ipynb]: Yeah I don't think it really matters if we leave extra cells at the end. But we may as well remove them if we find them.

File: course_content/iris_course/6.Data_Processing.ipynb (hunk 7 above)
Comment (lbdreyer) [reply course_content/iris_course/6.Data_Processing.ipynb]: Yeah, I guess "the data of a Cube" misses out the importance of metadata handling.
I will go with your suggestion

**Comment by lbdreyer** [comment]
@pp-mo  I have pushed up a new commit that hopefully addresses your review comments

File: course_content/iris_course/1.The_Iris_Cube.ipynb [hunk 13]
```diff
@@ -206,8 +208,9 @@
    "metadata": {},
    "source": [
     "<div class=\"alert alert-block alert-warning\">\n",
-    "    <b><font color='brown'>Exercise: </font></b> <p>What are the dimensions of the \"total electron content\" cube?</p>\n",
-    "    <p>What are the units of the \"electron_density\" cube?</p>\n",
+    "    <b><font color='brown'>Exercise: </font></b>\n",
+    "    <p>What are the dimensions of the \"total electron content\" cube?\n",
+    "    <br>What are the units of the \"electron_density\" cube?</p>\n",
```
Comment (lbdreyer) [inline course_content/iris_course/1.The_Iris_Cube.ipynb]: The difference between `<p></p>` and `<br>` is more obvious in jupyter lab than jupyter notebook.

I think `<br>` looks better.

Comment (pp-mo) [reply course_content/iris_course/1.The_Iris_Cube.ipynb]: Ok !

**Comment by pp-mo** [comment]
All good !

**Pull request merged**